### DISCUSSION TITLE
Can this library support absolute numbers and not x, y coordinates?

### DISCUSSION THREAD

**Discussion by sahilsian** [N1ghtly/react-native-responsive-linechart#62] [Q&A]
I want to be able to feed the chart an array similar to this:
data={[1, 2, 5, 2, 5, 10, 11]}

**Comment by xanderdeseyn** [comment]
How would that work exactly? Because to be able to plot on a 2D plane, you need two points haha

**Reply by sahilsian** [reply]
Well, this comes mostly from what react native chart kit does with their library. Stock API's aren't able to feed X/Y coordinates, the end goal I want to do is to be able to visualize stock quote changes into a graph. I'm currently not using react native chart kit just due to visual reasons that your library does really well on (love the area component).

React native chart kit: https://github.com/indiespirit/react-native-chart-kit

**Reply by sahilsian** [reply]
Actually nevermind, just realized that I could use time as an x value LOL

**Reply by sahilsian** [reply]
Is there any way to keep the tool tip in a fixed position on the graph?

**Reply by xanderdeseyn** [reply]
You can set initialTooltipIndex, and then set the disableGestures and disableTouch props on Chart
